people/nikola.json

```diff
@@ -0,0 +1,10 @@
+{
+    "key": "nikola",
+    "name": "Nikola Svobodová",
+    "nick": "Nikola",
+    "email": "nikola.svobodova@mild.blue",
+    "title_cs": "Backend developer",
+    "title_en": "Backend developer",
+    "short_desc_en": "While Nikola studies Computer Science at CTU and Mathematics at MFF UK, she has previously worked as a researcher at the experimental molbio-biochem laboratory. Currently, she is responsible for the backend development of our products.",
```

Comment:
Jses si jista tim While takhle pouzitym? Mozna ze je to nejaky clause, ktery takhle neznam, Nejaka formulace jako tohle nize by mi asi prisla lepsi, ale jestli je to while takhle pouzivany, rad se necham poucit :)

Nikola studies Computer Science at CTU and Mathematics at MFF UK. Apart from that, Nikola has also research experience from an experimental molbio-biochem laboratory

A molbio-biochem je fakt takhle pouzivana zkratka? (Chapu ze je to molekularni biologie, jen takhle tu zkratku neznam, tak se ujistuju, ze je obecne zname:)

Reply:
Diky za comment ;) Jak jsi to napsal, tak i mne se to libi vic. Muzeme to tam tak dat.

Ta zkratka molbio neni oficialni, ale normalne se pouziva. To muzeme asi radeji rozepsat na experimental biochemistry/molecular biology laboratory.